packages/MarkdownEditor-Core.package/MarkdownSettings.class/instance/menubarContent..st

```diff
@@ -0,0 +1,21 @@
+building
+menubarContent: aBuilder
+			
+	^ {aBuilder pluggableButtonSpec new
```

Comment:
Code redundancy: Maybe create a Collection {'General', 'Headings', 'Lists' } and iterate on it!